### PR TITLE
Fixed wrong exp values on 13.1 and 13.2 quests and other minor fixes

### PR DESCRIPTION
**Pull request by Daegaladh**
* **Addressed Issue(s)**: #3846 

* **Server Mode**: Both

* **Description of Pull Request**: Fixed 13.1 and 13.2 quests exp values according to aegis data.

### CODE REVIEW

File: npc/quests/quests_13_1.txt [hunk 1]
```diff
@@ -2922,7 +2922,7 @@ mid_campin,106,122,4	script	Receptionist Brink#ep13b	89,{
 	}
 }
 
-mid_camp,264,263,4	script	Diego#ep13bs	931,{
+mid_camp,266,263,4	script	Diego#ep13bs	931,{
```
Comment (Atemo) [inline npc/quests/quests_13_1.txt]:
```
//npc "mid_camp" "Diego#ep13bs" 4_M_RACHMAN1 264 263 4 0 0
npc "mid_camp" "Diego#ep13bs" 4_M_RACHMAN1 266 263 4 0 0
```
266 for renewal. Split the npc into re/pre-re

Comment (Daegaladh) [reply npc/quests/quests_13_1.txt]: Right, thanks!

File: npc/quests/quests_13_2.txt [hunk 2]
```diff
@@ -4803,7 +4803,7 @@ man_in01,372,221,5	script	Scientist#lifeguard	449,{
 				delitem 6078,.@stoneelse_chk; //Stone_Fragments
 				delitem 6076,1; //Portable_Toolbox
 				getitem 6080,3; //Manuk_Coin
-				getexp 0,(checkre(3))?50000:700000; //unconfirmed (RE)
+				//getexp 0,(checkre(3))?70000:700000; //iRO only
```
Comment (Lemongrass3110) [inline npc/quests/quests_13_2.txt]: This removes pre-renewal bonus as well?

File: npc/quests/quests_13_2.txt [hunk 3]
```diff
@@ -5671,8 +5671,7 @@ spl_in01,30,324,3	script	High Laphine#grenouille	446,{
 					set ep13_2_tre1,6;
 					changequest 2185,2186;
 					getitem 6081,3; //Splendide_Coin
-					if (checkre(3) == 0)
-						getexp 0,(checkre(3))?50000:700000; //unconfirmed (RE)
+					//getexp 0,(checkre(3))?70000:700000; //iRO only
```
Comment (Lemongrass3110) [inline npc/quests/quests_13_2.txt]: This removes pre-renewal bonus as well?

File: npc/quests/quests_13_1.txt [hunk 4]
```diff
@@ -8888,10 +8888,7 @@ mid_campin,90,121,5	script	Hibba Agip	459,{
 		mes "And please take this small reward.";
 		mes "Thank you for your help so far.";
 		next;
-		if (checkre(3))
-			getexp 300000,0;
-		else
-			getexp 2500000,0;
+		getexp (checkre(3))?300000:250000,0;
```
Comment (Lemongrass3110) [inline npc/quests/quests_13_1.txt]: Is that change really needed instead of simply removing one zero in pre-renewal?
This way you cannot see what was fixed clearly.

Comment (Daegaladh) [reply npc/quests/quests_13_1.txt]: No that's because it gives no job exp, so you can optimize it in that way, and also to maintain the slyle of the rest of the file.

File: npc/quests/quests_13_2.txt (hunk 3 above)
Comment (Daegaladh) [reply npc/quests/quests_13_2.txt]: Yeah, it's commented on both aegis renewal an pre-renewal
